server/submission/resolvers.js

```diff
@@ -48,9 +49,23 @@ async function setupCorrespondingAuthor(user, manuscript) {
     manuscript.manuscriptPersons.push(manuscriptPerson)
 
     if (!manuscriptPerson.user) {
+      const { email, firstName, lastName } = manuscript.submissionMeta.author
+      const { title, id } = manuscript
+
+      const emailToken = authentication.token.create({ username: email })
+
       await mailer.send({
-        to: manuscript.submissionMeta.author.email,
+        to: email,
         text: 'Please verify that you are a corresponding author',
+        html: `<p>${firstName} ${lastName} has recently submitted a paper to eLife, with the title "${title}". You have been listed as the corresponding author
```

Comment:
We should probably escape these values as they are user supplied.

Reply:
Looking at this package for santisation: https://www.npmjs.com/package/sanitize-html

Reply:
I think it would probably be enough to just convert `<` and `>` to `&lt;` and `&gt;`.